testing/test_aux.out

```diff
@@ -65,10 +65,12 @@ port-01 module_config modules
     "enabled": true
   },
   "nmap": {
-    "enabled": true
+    "enabled": true,
+    "timeout": 60
```

Comment:
I don't see anything where you actually test the timeout capability?  There should be something in here that is expected to timeout (e.g. you could go the other way and set a very short timeout on one of these enabled tests), and then explicitly check for that expected failure in the output results.  E.g., if the timeout code gets messed up for some reason, how would we know (which is the purpose of the integration tests).